tests/test_arrow.py

```diff
@@ -189,7 +190,39 @@ def test_repr(self):
 
         result = self.arrow.__repr__()
 
-        assert result == f"<Arrow [{self.arrow._datetime.isoformat()}]>"
+        assert result == "arrow.Arrow(2013, 2, 3, 12, 30, 45, 1, tzinfo=tzutc())"
+
+    @pytest.mark.skipif(
```

Comment:
Also I think it might be better if we just combine these into one test and dynamically obtain the tzinfo string by doing something like this: `tz_str = str(tz.gettz("Europe/Stockholm"))`. This will dynamically generate the path depending on the OS.